BaseFiles/Common/html/pages/events/_events.js

```diff
@@ -263,13 +263,20 @@ HG.WebApp.Events.SendEventToUi = function (module, eventLog) {
                     if (group != null) group = group.Name;
                     var name = module.Domain.substring(module.Domain.indexOf('.') + 1) + ' ' + module.Address;
                     var propname = eventLog.Property.substring(eventLog.Property.indexOf('.') + 1);
-                    var value = (parseFloat(eventLog.Value.replace(',', '.')).toFixed(2));
-                    // TODO: should level be reported as is??
-                    if (propname == 'Level') {
-                        value = value * 100;
-                        if (value > 98) value = 100;
-                    }
-                    value += '%';
+                    
+                    if(propname == "DoorLock"){
```

Comment:
(self-note) instead of adding any other specific code, will add checking of eventLog.Value type, and if it is numeric it will be processed (eg. toFixed), otherwise it will be left as is.